.dockerignore

```diff
@@ -0,0 +1,2 @@
+scripts/minio-conf/
+scripts/minio-data/
```

Comment:
As this PR is related to minio directory changes. Can you please remove this line - [here](https://github.com/RedHatInsights/ros-backend/blame/main/scripts/README.md#L14) from the Readme